docs/avs/running-avs-operator-mainnet.md

```diff
@@ -125,18 +123,33 @@ ATTESTATION_CENTER_ADDRESS=0x2B766957ce3dbab9eC4b227f5086855CeE7a1ad6
-| ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `PRIVATE_KEY`       | The [Consensus Key](https://docs.othentic.xyz/main/avs-framework/othentic-cli/private-key-management#consensus-key)                                                                      |
-| `OPERATOR_ADDRESS`  | The public address of the [Controller Key](https://docs.othentic.xyz/main/avs-framework/othentic-cli/private-key-management#controller-key) (optional if both private keys are the same) |
-| `L1_RPC` / `L2_RPC` | The RPC endpoint addresses for the L1 (Ethereum) and L2 (Base) networks, respectively                                                                                                    |
+| Parameter             | Description                                                                                                                                                                              |
+| --------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `PRIVATE_KEY`         | The [Consensus Key](https://docs.othentic.xyz/main/avs-framework/othentic-cli/private-key-management#consensus-key)                                                                      |
+| `OPERATOR_ADDRESS`    | The public address of the [Controller Key](https://docs.othentic.xyz/main/avs-framework/othentic-cli/private-key-management#controller-key) (optional if both private keys are the same) |
+| `L1_RPC` / `L2_RPC`   | The RPC endpoint addresses for the L1 (Ethereum) and L2 (Base) networks, respectively                                                                                                    |
+| `ANNOUNCED_ADDRESSES` | see frame below for details                                                                                                                                                              |
 
 :::tip
 
 - The fields marked with comments (#) need to be filled in with the appropriate values specific to the operator’s setup.
 - Instead of using a `.env` file, you can set these values as system-wide environment variables.
   :::
 
+:::tip
+If your docker container has IP address that is publicly available leave `--announced_addresses` and `$ANNOUNCED_ADDRESSES` parameters in `docker-compose.yml` commented out.
+Otherwise, to make sure your node can be connected to from our aggregator node you need to provide your public address in a form of multi-address [details](https://docs.libp2p.io/concepts/fundamentals/addressing/).
+You will need to fill-in `ANNOUNCED_ADDRESSES` variable if `docker-compose.yml`
```

Comment:
this sentence looks weird/unfinished?

Reply:
<img width="864" alt="image" src="https://github.com/user-attachments/assets/b9813d5d-bb8a-4d7e-a287-07078c08b15b" />
Please don't hate me, but there's probably still missing `.` before the selected `If`.

Reply:
Also, I would start each "If" (i.e. `If your node is avaialbe via IP` and `If, on the other hand`) from a new line, to sligtly increase the readability.